lib/tasks/db.rake

```diff
@@ -0,0 +1,49 @@
+namespace :db do
+
+  desc 'Dumps the database to file'
+  task :dump => :environment do
+
+    host = ActiveRecord::Base.connection_config[:host]
+    db = ActiveRecord::Base.connection_config[:database]
+
+    cmd = "pg_dump --no-owner --no-acl -v -f #{dump_file} --no-password "
```

Comment:
dump_file is a private function further down, so it could shard between the two tasks. I'll add -c.